examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3631,6 +3635,10 @@ class GLTFParser {
 			const inverseBindMatrices = results.pop();
 			const jointNodes = results;
 
+			// Note that bones (joint nodes) may or may not be in the
+			// scene graph at this time.
+			// Child nodes are added in .loadNode() (no '_' prefix).
```

Comment:
The comment might be a little misleading because this function, .loadSkin, has just been invoked by .loadNode, but not necessarily the same .loadNode call that is going to populate the joints of this skin. I would probably just remove the second sentence, "Child nodes are..." and try to clarify the method names instead. How about this?

- `.loadNode`
- `.loadNodeShallow`

With or without the underscore is fine with me!

Reply:
Thanks for renaming suggestion. I know I'm bad at naming in English so renaming suggestion is always helpful to me.

I renamed `._loadNode` to `._loadNodeShallow`. I'm not sure we should really expose the method from the parser so didn't remove the underscore but we may rename later if users request.

Reply:
Doesn't need to be a public method, agreed! :)